server/app/services/cloud/aws/SimpleStorage.java

```diff
@@ -50,8 +52,8 @@ public SimpleStorage(
       ApplicationLifecycle appLifecycle) {
     this.region = checkNotNull(region).get();
     this.credentials = checkNotNull(credentials);
-    this.bucket = checkNotNull(config).getString(AWS_S3_BUCKET_CONF_PATH);
-
+    this.config = checkNotNull(config);
```

Comment:
Not blocking: I'd either store the file limit as it's own var and initialize it here or remove "bucket" as a field and read it's value from the config in a similar way.

Reply:
done.